.murdock

```diff
@@ -137,7 +147,12 @@ compile() {
     local board=$(echo $2 | cut -f 1 -d':')
     local toolchain=$(echo $2 | cut -f 2 -d':')
 
-    [ "$board" = "makefile_broken" ] && error "$0: Makefile in \"$appdir\" seems to be broken!"
+    [ "$board" = "makefile_broken" ] && {
+        echo "$0: There seems to be a problem in \"$appdir\" while getting supported boards!"
+        echo "$0: testing \"make -C$appdir make-info-boards-supported\"..."
+        make -C$appdir make-info-boards-supported && echo "$0: success. no idea what's wrong." || echo "$0: failed!"
```

Comment:
is this right? I cannot find a target `make-info-boards-supported` just `info-boards-supported`:

```
git grep make-info-boards-supported

```

but 
```
git grep info-boards-supported
.murdock:    local boards="$(make --no-print-directory -C$appdir info-boards-supported 2>/dev/null || echo broken)"
Makefile.include:GLOBAL_GOALS += buildtest info-boards-supported info-boards-features-missing info-buildsizes info-buildsizes-diff
dist/tools/compile_and_test_for_board/compile_and_test_for_board.py:        cmd = ['info-boards-supported']
makefiles/info-global.inc.mk:info-boards-supported:
makefiles/info.inc.mk:.PHONY: info-objsize info-buildsizes info-build info-boards-supported \
makefiles/info.inc.mk:  @echo $$($(MAKE) info-boards-supported)
```

Reply:
uff, well spotted. this didn't show up in my test case as that broke the syntax of the makefile, not even trying to build the (non-existing) target "make-info-boards-supported".

fixed.